aea/configurations/base.py

```diff
@@ -243,7 +243,7 @@ def from_json(cls, obj: Dict):
             restricted_to_protocols=cast(Set[str], restricted_to_protocols),
             excluded_protocols=cast(Set[str], excluded_protocols),
             dependencies=dependencies,
-            description=cast(str, obj.get("description")),
+            description=cast(str, obj.get("description", "")),
```

Comment:
without the default value `""`, the default was `None` and this caused issues when validating the configuration in json format, since it expeted a `"string"` as per JSON schema.